support/cas-server-support-actions/src/main/java/org/apereo/cas/web/config/CasSupportActionsConfiguration.java

```diff
@@ -273,4 +274,9 @@ public Action serviceWarningAction() {
         return new ServiceWarningAction(centralAuthenticationService, authenticationSystemSupport,
             ticketRegistrySupport, warnCookieGenerator.getIfAvailable(), principalElectionStrategy);
     }
+
+    @Bean
+    public Action mfaInitializeAction() {
```

Comment:
Also not sure if this belongs here. Shouldn't the MFA-related actions, etc be housed in a module that is only required for MFA, and not by default?

Reply:
Should we do the same with cas-sever-core-webflow-api, that has MFA implementations?  Create cas-server-core-webflow-mfa and cas-server-core-weblfow-mfa-api and refactor all MFA classes to these two?